ruka/velero/velero.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/env bash
+
+#confirm these variables before installing
+NAMESPACE='velero'
+BUCKET='it-backup-test'
+BKP_NAME='default'
+VOL_NAME='default'
+PREFIX='ruka'
+S3Url='https://it-s3.ls.lsst.org'
+RGW_SECRET='it-s3-credentials'
+REGION='default'
+
+echo -e "\nPlease review these values before installing velero"
+echo "namespace: $NAMESPACE"
+echo "prefix: $PREFIX"
+echo "bucket: $BUCKET"
+echo "S3 Url: $S3Url"
+echo "Are these values correct? (Y/N): "
+while true; do
+    read -r input
```

Comment:
I find scripts which prompt for input with no-way to bypass it frustrating to use.

Reply:
It's a fair argument but prevents mistakes. What do you think @cbarria?

Reply:
i would use ENV variables, but checking in the script is they exist first instead of using a user input... specially thinking in the future when deployments get automated =)

Reply:
Ok, I will remove it.